internal/xds/translator/httpfilters.go

```diff
@@ -37,14 +38,16 @@ func newOrderedHTTPFilter(filter *hcmv3.HttpFilter) *OrderedHTTPFilter {
 	order := 50
 
 	// Set a rational order for all the filters.
-	switch filter.Name {
-	case wellknown.CORS:
+	switch {
+	case filter.Name == wellknown.CORS:
 		order = 1
-	case jwtAuthnFilter:
+	case filter.Name == jwtAuthnFilter:
```

Comment:
shouldn't this be after Oauth2 ?

Reply:
case isOAuth2Filter(filter):

Reply:
Should we put `HTTPRateLimit` at the first?